website/docs/src/content/component-reference/storage-components/blob-reference.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Blob storage reference
+description: Blob storage components for stateful workflows and agents.
+---
+
+# Blob Storage Reference
+
+API reference for GenSX Cloud blob storage components.
+
```

Comment:
Probably worth adding a super short "installation" section to this and the other storage reference docs to be consistent with all the other component reference docs